src/snakemake/checkpoints.py

```
@@ -62,18 +68,25 @@ def get(self, **wildcards):
-        if self.checkpoints.created_output:
-            missing_output = set(output) - set(self.checkpoints.created_output)
-            if not missing_output:
-                return CheckpointJob(self.rule, output)
-            else:
-                logger.debug(
-                    f"Missing checkpoint output for {self.rule.name} "
-                    f"(wildcards: {wildcards}): {','.join(missing_output)} of {','.join(output)}"
-                )
-
-        raise IncompleteCheckpointException(self.rule, checkpoint_target(output[0]))
+        listify = lambda x: [x,] if not isinstance(x, list) else x
```

Ruff (0.8.2)

Do not assign a `lambda` expression, use a `def`

Rewrite `listify` as a `def`

(E731)